el-patch.el

```diff
@@ -436,14 +436,18 @@ is a sentence to put in brackets at the end of the docstring."
          type))
       (when (and docstring-note docstring-idx)
         (let ((old-docstring (nth docstring-idx definition)))
-          (when (stringp old-docstring)
-            (let ((new-docstring
-                   (concat
-                    old-docstring
-                    (format "\n\n[%s]" docstring-note))))
-              (setq definition (cl-copy-list definition))
-              (setf (nth docstring-idx definition)
-                    new-docstring)))))
+		  (setq definition (cl-copy-list definition))
```

Comment:
Please turn off `indent-tabs-mode`. In fact, I recommend doing this globally, since [almost everyone agrees it should never be used](https://github.com/bbatsov/emacs-lisp-style-guide).

Reply:
Are my changes correct?